src/ctia/http/routes/properties.clj

```diff
@@ -1,10 +1,14 @@
 (ns ctia.http.routes.properties
   (:require [compojure.api.sweet :refer :all]
+            [ctia.properties :refer [properties]]
             [ring.util.http-response :refer :all]
-            [ctia.properties :refer [properties]]))
+            [schema.core :as s]))
 
 (defroutes properties-routes
   (context "/properties" []
-    :tags ["Properties"]
-    (GET "/" []
-      (ok @properties))))
+           :tags ["Properties"]
+           :summary "The currently running properties"
+           :capabilities :admin
+           :header-params [api_key :- (s/maybe s/Str)]
```

Comment:
This type information is not in the context of a schema.core macro. So the `:-` symbol and the `(s/maybe s/Str)` expression both end up being treated as allowable params. Of course, these will never be passed, so they are essentially ignored. But they're not type info, and they're not comments, so they shouldn't be there in this form.